### DISCUSSION TITLE
Removing Bookmarks on the New Tab (View Image)

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1286] [Ideas]
Ungoogled-Chromium's NewTab looks like this:
![image](https://user-images.githubusercontent.com/52204908/102057083-a9307200-3ded-11eb-8251-8f9e982be6e6.png)

It would be good if there was a Flag for removing those Bookmarks or Recently Viewed Web Sites. I know there are extensions to make the homepage Black, but I was trying to avoid those.

For now I am using this Extension: https://chrome.google.com/webstore/detail/blank-dark-new-tab-page/agldajbhchobfgjcmmigehfdcjbmipne

**Comment by rry6m0ejl3j** [comment]
There actually is a flag to hide the bookmarks bar on the new tab page.
- Go to `chrome://flags`.
- Search for `bookmark-bar-ntp`. There should be one result.
- Change the setting from `Default` to `Never`.

Alternatively, use the command-line switch `--bookmark-bar-ntp=never`.

**Reply by ghost** [reply]
Thanks, It worked!

**Reply by ghost** [reply]
Update - I noticed now that this does **NOT** remove the bookmarks on the Home Page, but it removes the top bar. It doesn't have anything to do with my problem.

**Reply by Ahrotahn** [reply]
Adding `--disable-top-sites` to your launch switches should disable the new tab tiles.
